drivers/TSP01/driver.js

```diff
@@ -0,0 +1,265 @@
+'use strict';
+
+const path = require('path');
+const ZwaveDriver = require('homey-zwavedriver');
+
+// http://www.tkbhome.com/?cn-p-d-258.html
+// http://www.philio-tech.com/pdf/PSP01.pdf
+// https://www.intellihome.be/nl/z-wave-tkb-3-in-1-sensor-pir-light-temperature.html
+// TSP01
+
+module.exports = new ZwaveDriver( path.basename(__dirname), {
+	capabilities: {
+		'alarm_motion': {
+			'resetTimer'                : 'motion_inactivity_timer',	// Can reference to a device setting or a number.
+			'resetTimerCallback'        : function (capabilityRef, refNo, node) {
+				if (capabilityRef.refNo == refNo) {
```

Comment:
true, it has been a while :). In my mind a === is extra strict and also would check if it is the same memory reference, instead of a comparison where the number would match...